.github/trusted-contribution.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+annotations:
+  - type: comment
+    text: "/gcbrun"
+trustedContributors:
+  - release-please[bot]
+  - renovate[bot]
+  - forking-renovate[bot]
+  - dependabot[bot]
+  - apeabody
```

Comment:
@apeabody - Can we try replacing username with `@GoogleCloudPlatform/point-of-sale`?

Reply:
Also can you accept the invite for the 2 groups I've sent?